package.json

```diff
@@ -2,7 +2,7 @@
   "name": "vscode-ytmusic",
   "displayName": "vscode-ytmusic",
   "description": "Connecting VS Code with the YTMDesktop Player",
-  "version": "1.2.2",
+  "version": "1.2.3",
```

Comment:
Was thinking that this feature would probably be warrant a minor bump so it'd be 1.3.0. Ideally version bumps should be in their own commit, feel free to either do as part of this PR, or switch back to 1.2.2 and I can add a commit once merged :)

Reply:
alright, updated it as part of this PR